adapters/socketio_berkeley.c

```diff
@@ -569,8 +716,6 @@ void socketio_destroy(CONCRETE_IO_HANDLE socket_io)
 int socketio_open(CONCRETE_IO_HANDLE socket_io, ON_IO_OPEN_COMPLETE on_io_open_complete, void* on_io_open_complete_context, ON_BYTES_RECEIVED on_bytes_received, void* on_bytes_received_context, ON_IO_ERROR on_io_error, void* on_io_error_context)
```

Comment:
>socketio_open [](start = 4, length = 13)

You forgot to fix the tests for this and add a couple for the new functionality.  Testing for this unit is very hit and miss.  I'd like to put in on my backlog to add more.